From 230a376b5a67f0e9341e1fa47e670ff762213c83 Mon Sep 17 00:00:00 2001
From: Brad Fitzpatrick <bradfitz@golang.org>
Date: Mon, 12 Dec 2016 17:35:25 +0000
Subject: [PATCH 38/38] [release-branch.go1.7] doc: remove Reddit as an
 official space where Code of Conduct applies

Fixes #18289

Change-Id: I047e98cc36b861ef15292170aeaff8bc29243cab
Reviewed-on: https://go-review.googlesource.com/34281
Reviewed-by: Russ Cox <rsc@golang.org>
(cherry picked from commit 265e547658ac6e91df308326041ea462593053fc)
Reviewed-on: https://go-review.googlesource.com/34282
---
 doc/conduct.html | 1 -
 1 file changed, 1 deletion(-)

diff --git a/doc/conduct.html b/doc/conduct.html
index ce824a1..d4d0800 100644
--- a/doc/conduct.html
+++ b/doc/conduct.html
@@ -67,7 +67,6 @@ official forums operated by the Go project (“Go spaces”):
     <li>The <a href="https://groups.google.com/group/golang-nuts">golang-nuts</a> and
         <a href="https://groups.google.com/group/golang-dev">golang-dev</a> mailing lists.
 <li>The #go-nuts IRC channel on Freenode.
-<li>The <a href="https://reddit.com/r/golang">/r/golang subreddit</a>.
 </ul>
 
 <p>
-- 
2.7.4

